test/parallel/test-process-env-ignore-getter-setter.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+
+assert.throws(
+  () => {
+    Object.defineProperty(process.env, 'foo', {
+      value: 'foo1'
+    });
+  },
+  {
+    code: 'ERR_INVALID_OBJECT_DEFINE_PROPERTY',
+    name: 'TypeError',
+    message: 'Must set all attributes with true to \'value\' ' +
```

Comment:
this message is confusing to me - process.env values should always be strings or undefined, never booleans.

Reply:
changed to `'process.env' only accepts a configurable, writable and enumerable data property`